bootspec/src/v1.rs

```diff
@@ -112,7 +112,11 @@ impl BootSpecV1 {
                 err: e,
             })?;
 
-        let kernel_file = generation.join("kernel-modules/bzImage");
+        let kernel_image_name = match system.as_str() {
+            "x86_64-linux" => "bzImage",
```

Comment:
I wonder if this should also be `i686-linux`, but I imagine there are so few systems out there running that that we can revisit this later if someone notices it's broken.

Reply:
true, i686-linux should use `bzImage` too.